code/game/objects/items.dm

```diff
@@ -406,7 +410,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 			M.eye_blurry += 10
 			M.Paralyse(1)

```

Comment:
You've removed an if check I added that made sure the person wasn't already paralysed or weakened before applying the stun. You currently have a 50% success rate of chainstunning someone.